docs/content/preview/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -16,6 +16,40 @@ menu:
 type: docs
 ---
 
+## Replace a live or unreachable node
+
+If a node (live or unreachable) needs to be replaced, do the following:
+
+1. Navigate to **Universes**, select your universe, and open the **Nodes** tab.
+1. Click the associated node **Actions > Replace Node**.
+
+    ![Replace Node Actions](/images/ee/replace-node.png)
+
+Clicking the **Replace Node** button opens a popup window to confirm the replacement of this node. Confirming the task redirects to the task page and will take a few minutes to complete.
```

Comment:
I don't think confirming replacing redirects to task page, it should reload the page and the loading circle and the progress bar should slowly run

@rajmaddy89  can you confirm?

Reply:
I have just added this instead and I think this should suffice - 
1. Click OK to confirm.
YugabyteDB Anywhere (YBA) starts the node replacement process, and you can view the progress on the **Tasks** tab.